### PR TITLE
Test installing all operators in repository

### PR DESCRIPTION
**Pull request by nfnt**
This also includes demo operators in package documentations.

### CODE REVIEW

File: kudo-test.yaml
Comment (mpereira) [inline kudo-test.yaml:10]:
```suggestion
- kudo install --skip-instance ./repository/cassandra/3.11/operator/
- kudo install --skip-instance ./repository/cowsay/operator/
```

**Review by kensipe — APPROVED** [review]
merged to the latest on master... waiting for green otherwise lgtm

**Comment by nfnt** [comment]
Re-requesting reviews, because the removal of Kubernetes 1.15 testing was done after @kensipe's approval.

**Review by ANeumann82 — APPROVED** [review]
Waiting for green, but otherwise lgtm

**Comment by ANeumann82** [comment]
Oh, it's only DCO missing, therefore, go fo rit

**Comment by nfnt** [comment]
I'll sort out the DCO issue when squashing.

**Review by zmalik — APPROVED** [review]
LGTM!

**Pull request merged**
